### ISSUE TITLE
Missing headers in phpmyadmin 5.1 

### ISSUE THREAD

**Issue by dark-phoenix009** [AIOFrame/phpMyAdmin#2]
Can this be updated for phpmyadmin 5.1? It looks broken and weird while most of the UI remains as it is
![image](https://user-images.githubusercontent.com/25230903/119227013-969f2f00-bb29-11eb-8703-0717785fdfa5.png)

**Comment by AIOFrame** [comment]
Yes, I am aware of this release and tried to quickly do some fixes but they completely changed the structure and I might have to spend a full day on this and I wasn't getting that time. Ill try to do this in a week or two.

**Comment by futurusticdev** [comment]
It's October mate, any update? The theme is great i love it but can't use it at this state. Bummer.

**Comment by tecnoweb** [comment]
can you tell me how to fix? ... i love this theme

**Comment by KostaDokosic** [comment]
any updates on this?

**Comment by AIOFrame** [comment]
Please give me till 24th of this month. I will have to re-design a new theme as phpmyadmin completely changed their html structure. I sincerely apologize for the inconvenience.

**Comment by berkayaksoy** [comment]
There are still compatibility problems with version 5.1.1. The theme is very beautiful and we are waiting for the update. @AIOFrame

**Comment by anthonyrpitts** [comment]
Hello, love the theme and would love to use it full time, however the headers are covering information, 
relating the settings of the server.  Thank you for your awesome work :-)

Server Configuration:

- Apache/2.4.51 (Win64) 
- OpenSSL/1.1.1l 
- Database client version: libmysql - mysqlnd 7.4.26
- PHP extension: mysqli curl mbstring 
- PHP version: 7.4.26
- **phpMyAdmin Version information: 5.1.1** (up to date)

![aiophp](https://user-images.githubusercontent.com/95981408/145688991-f57a24bb-1e72-4629-96e2-b13f4160eaec.png)

**Comment by mabuxi** [comment]
Hello do you have an update? The problem still here in november 2022

**Comment by alfari24** [comment]
I have tried several versions of phpmyadmin from version 4.9 - 5.2
and the lates theme can **only** be used for **version 5.0.4**. and still not **compatible with 5.1**
